sdk/eventhub/Azure.Messaging.EventHubs/tests/Amqp/AmqpEventBatchTests.cs

```diff
@@ -9,10 +9,10 @@
 using Azure.Messaging.EventHubs.Core;
 using Azure.Messaging.EventHubs.Producer;
 using Microsoft.Azure.Amqp;
-using Microsoft.Azure.Amqp.Framing;
 using Moq;
 using NUnit.Framework;
 using NUnit.Framework.Constraints;
+using FramingData = Microsoft.Azure.Amqp.Framing.Data;
```

Comment:
(extreme) nit:  Can we throw a blank line between the "I'm using these things" `using` statement and the "Alias this` `using` statement?

Reply:
Added. :-)